learning-progression.md

```diff
@@ -164,22 +172,32 @@ if (false) {
 
 ### Step 6: Data types
 
+#### 1. Study
+
+The basic functions of a computer are to store and process electrical signals which represent binary numbers, 1's and 0's. These numbers are used to represent all of the information in our computers, from a simple text document to an HD movie.
```

Comment:
Immediate ambiguity with the computer-programming term "function". Human language _a priori_ ambiguous, but engineering materials should not be. Neither are programming languages.